Resources/public/hbt/default.hbt

```diff
@@ -0,0 +1,29 @@
+<div class="content-view-line">
+    {{#each recommendations}}
+    <article class="class-article">
```

Comment:
this is based on what we did in legacy (class == contentType), I guess @dpobel can provide some info on what we should use now. DemoBundle and PlatformUi has some conventions already, but I suspect we should define this somewhere in our coding guidelines for html.